### PR TITLE
Issue 196: Fix multiple additions of images to DOM

### PR DESCRIPTION
**Pull request by tmathern**
Fix https://github.com/hlxsites/caesars/issues/196

## 🔗 Test URLs:
  
- Before: https://main--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/issue-196
- After: https://issue-196--caesars--hlxsites.hlx.page/caesars-palace/drafts/tmathern/issue-196

## 📝 Description:
  
Repro: The issue can be reproduced by changing window size with the pointer. On size change switch mobile/desktop, images will be added to the DOM/

### CODE REVIEW

**Comment by amol-anand** [comment]
This is brutal as you already mentioned 😄 
Hold off on merging this. It should not be this complicated. Is this because they want a totally different image in mobile or that the ootb sizing is not sufficient? you can also use getOptimizedPicture() and pass in the right width for each breakpoint etc.. 
And this has gotten to a point where it no longer should be in the default/ootb columns block. Might be better off as a separate block instead.

**Review by amol-anand — CHANGES_REQUESTED** [review]
Hold off on merging this. It should not be this complicated. Is this because they want a totally different image in mobile or that the ootb sizing is not sufficient? you can also use getOptimizedPicture() and pass in the right width for each breakpoint etc..

File: caesars-palace/blocks/columns/columns.js [hunk 1]
```diff
@@ -44,15 +45,4 @@ export default function decorate(block) {
       mqList.addEventListener('change', handleScreenChange);
     }
   }
-
-  const mediaQueryMatcher = window.matchMedia('(max-width: 768px)');
-  const handleScreenChange = (query) => {
-    if (query.matches) {
-      block.querySelectorAll('img').forEach((image) => {
-        image.closest('picture').replaceWith(createOptimizedPicture(image.src, image.alt, false, [{ width: '768' }]));
```
Comment (tmathern) [inline caesars-palace/blocks/columns/columns.js:52]: Why did we need this here explictly again?

Comment (tmathern) [reply caesars-palace/blocks/columns/columns.js:52]: The bug occurs only if using "replaceWith". I suspect it has to do something with the fact that replaeWith will do some cloning (hence we'll loose the original `picture` reference we got through the closure). I tried putting an id on the picture to grab it by id and move it, but a duplicate still managed to appear. So if we don't need all this, it's probably best to drop it.

**Review by swetabar — APPROVED** [review]
LGTM!

**Pull request merged**